source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRDateTextField.mm

```diff
@@ -0,0 +1,145 @@
+//
+//  ACRDateTextField
+//  ACRDateTextField.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRDateTextField.h"
+#import "DateInput.h"
+#import "TimeInput.h"
+
+using namespace AdaptiveCards;
+
+@implementation ACRDateTextField
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    return self;
+}
+- (instancetype)initWithTimeDateInput:(std::shared_ptr<BaseInputElement> const &)elem
+                            dateStyle:(NSDateFormatterStyle)dateStyle
+{
+    self = [self initWithFrame:CGRectMake(0,0,0,0)];
+
+    if(self)
+    {
+        NSString *placeHolderStr = nil;
+        NSString *minDateStr = nil;
+        NSString *maxDateStr = nil;
+        
+        NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+        formatter.dateStyle = dateStyle;
+        formatter.locale = [[NSLocale alloc] initWithLocaleIdentifier:@"en_US"];
```

Comment:
>en_US [](start = 72, length = 5)

Are you sure you always want to use US-style dates, rather than the user's own locale?